snap/info.go

```diff
@@ -65,8 +72,13 @@ type SlotInfo struct {
 type AppInfo struct {
 	Snap *Info
 
-	Name    string
-	Command string
-	Plugs   map[string]*PlugInfo
-	Slots   map[string]*SlotInfo
+	Name        string
+	Command     string
+	Description string
```

Comment:
Can we please take Description out from here until we make some better usage of it?  The current usage seems to be just for systemd services, which doesn't justify asking the user to type this in for every snap. The GUI is using the .desktop files.